medicines/web/.env.example

```diff
@@ -5,7 +5,3 @@ AZURE_SEARCH_KEY=
 AZURE_SEARCH_SCORING_PROFILE=
 AZURE_SEARCH_SERVICE=
 AZURE_SEARCH_WORD_FUZZINESS=
-GOOGLE_GTM_CONTAINER_ID=
```

Comment:
Have these env vars been deliberately removed?

Reply:
Yeah - these should stay!